evennia/settings_default.py

```diff
@@ -34,6 +34,8 @@
 # Lockdown mode will cut off the game from any external connections
 # and only allow connections from localhost. Requires a cold reboot.
 LOCKDOWN_MODE = False
+# Enables new account registration
+REGISTER_ENABLED = True
```

Comment:
This settings name is a bit ambivalent. I suggest being more explicit, like `NEW_ACCOUNT_REGISTRATION_ENABLED`